integration/spark/src/main/scala/org/apache/spark/sql/CarbonSqlParser.scala

```diff
@@ -1342,4 +1345,9 @@ class CarbonSqlParser()
       }
     }
 
+  protected lazy val dropDatabaseCascade: Parser[LogicalPlan] =
+    DROP ~> (DATABASE|SCHEMA) ~> opt(IF ~> EXISTS) ~> ident ~> CASCADE <~ opt(";") ^^ {
+      case cascade => throw new MalformedCarbonCommandException(
+          "Unsupported cascade operation in drop database command")
```

Comment:
since system supports both database and schema better to provide message including schema like database/schema command like "Unsupported cascade operation in drop database/schema command" , else better provide  "Unsupported cascade operation in drop command".